commons-oauth/src/main/java/eu/europeana/api/commons/oauth2/model/impl/EuropeanaApiCredentials.java

```diff
@@ -25,16 +26,22 @@ public EuropeanaApiCredentials(String userName, String clientId) {
     this.clientId = clientId;
 
   }
-  public EuropeanaApiCredentials(String userName, String clientId,String apikey) {
+  public EuropeanaApiCredentials(String userName, String clientId,String apikey, String affiliation) {
```

Comment:
this might break other APIs, the costructor without affiliation must still be available, as this is not a mandatory field